lib/cli/writer.js

```diff
@@ -67,6 +69,15 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
     return fs.openAsync(drive.raw, 'rs+');
   }).then((driveFileDescriptor) => {
     return imageStream.getFromFilePath(imagePath).then((image) => {
+      if (!image.size.final.estimation && !constraints.isDriveLargeEnough(drive, {
+        size: image.size.final.value
+      })) {
+        throw errors.createUserError(
+          'The image you selected is too big for this drive',
+          'Plug a bigger drive and try again'
```

Comment:
I fixed the "plug in" thing, but I"m not sure about drive/device. I guess it depends on which one users feel more comfortable with. In any case, we're using drive all across the code base, to probably not something we should handle here; we can send a PR that replaces drives with devices across the code base later.